num1 =2
num2 =0



try:
    x=num1/num2
except:
    print("error en el programa")

finally:
    print("programa de try")

